Projects/App/Sources/Present/Auth/SignUp/RegistrationPhoneNum/ViewController/RegistrationPhoneNumberViewController.swift

```diff
@@ -95,17 +87,25 @@ final class RegistrationPhoneNumberViewController: BaseVC<RegistrationPhoneNumbe
     }
     
     private func bindUI() {
+        let maxLength = 4
+        
         certificationNumberTextfield.rx.text.orEmpty
             .map { $0.count == 4 }
             .bind(with: self) { owner, isValid in
-                owner.nextButton.backgroundColor = isValid ? .black : ChoiceAsset.Colors.grayVoteButton.color
+                owner.nextButton.backgroundColor = isValid ? .black : SharedAsset.grayVoteButton.color
                 owner.nextButton.isEnabled = isValid
             }.disposed(by: disposeBag)
         
+        certificationNumberTextfield.rx.text.orEmpty
+            .map { $0.prefix(maxLength) }
+            .bind(with: self) { owner, text in
+                owner.certificationNumberTextfield.text = String(text)
+            }.disposed(by: disposeBag)
+        
```

Comment:
이 두 코드를 이렇게 리펙토링 해보면 어떨까요? 한 번의 bind로 줄일 수 있을 것 같습니다!
```suggestion
        certificationNumberTextfield.rx.text.orEmpty
            .map { text -> (Bool, String) in
                let isValid = text.count == 4
                let truncatedText = String(text.prefix(maxLength))
                return (isValid, truncatedText)
            }
            .bind(with: self, onNext: { owner, result in
                let (isValid, text) = result
                owner.nextButton.backgroundColor = isValid ? .black : SharedAsset.grayVoteButton.color
                owner.nextButton.isEnabled = isValid
                owner.certificationNumberTextfield.text = text
            }).disposed(by: disposeBag)
```

Reply:
이런 방식도 있군요 ! 감사합니다 🤓